auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -502,7 +502,7 @@ export default defineComponent({
     }
 
     const getRequestForAuthorizationStatusText = (affiliationInviteInfos: AffiliationInviteInfo[]) => {
-      if (isCurrentOrganization(affiliationInviteInfos[0].toOrg.id)) {
+      if (isCurrentOrganization(affiliationInviteInfos[0].toOrg?.id)) {
```

Comment:
What if it is an empty affiliation invitation list? is this checked elsewhere before invoked?

Reply:
```
          <span
            v-if="!!item.affiliationInvites"
            id="affiliationInvitesStatus"
          >
            <p style="font-size: 12px">
              <v-icon
                x-small
                color="primary"
              >mdi-account-cog</v-icon>
              <span v-html="getRequestForAuthorizationStatusText(item.affiliationInvites)" />
            </p>
          </span>
        </template>
        ```

Reply:
Looks like it's checked there